app/models/author.rb

```diff
@@ -1,6 +1,6 @@
 class Author < ApplicationRecord
-  has_many :bookauthors
-  has_many :books, through: :bookauthors
+  has_many :book_authors
+  has_many :books, through: :book_authors
```

Comment:
I wonder why it needs to be names with the underscore. I thought we could call it anything, like "frank" or something. Wonder if we just had inconsistencies.